pkgs/development/python-modules/cyclonedds-python/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+, cyclonedds
+, setuptools
+, rich-click
+
+, pytestCheckHook
+, pytest-mock
+}:
+
+buildPythonPackage rec {
+  pname = "cyclonedds-python";
+  version = "0.10.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "eclipse-cyclonedds";
+    repo = "cyclonedds-python";
+    rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```
help to avoid tag/branch confusion